editor/project_manager.cpp

```diff
@@ -2704,7 +2705,13 @@ ProjectManager::ProjectManager() {
 		// Fade the version label to be less prominent, but still readable.
 		version_btn->set_self_modulate(Color(1, 1, 1, 0.6));
 		version_btn->set_underline_mode(LinkButton::UNDERLINE_MODE_ON_HOVER);
-		version_btn->set_tooltip(TTR("Click to copy."));
+		String build_date;
```

Comment:
Maybe in the future this show be extracted into a separate class e.g. `EditorVersionLabel` to reduce code duplication?

Reply:
That makes sense, but I'd say it would be better done in a future PR (which I can handle).